src/Core/Feature.js

```diff
@@ -132,6 +132,12 @@ export class FeatureGeometry {
             }
         }
     }
+
+    setParentStyle(parentStyle) {
+        if (this.properties.style && this.properties.style.isStyle && !this.properties.style.parent.isStyle) {
```

Comment:
is necessary to check `isStyle`?

Reply:
It is not, I removed it, before we needed to do that but indeed it is not the case anymore.